api/models.py

```diff
@@ -292,6 +293,42 @@ def save(self, *args, **kwargs):
         super(AssetStatus, self).save(*args, **kwargs)
 
 
+class AllocationHistory(models.Model):
+    asset = models.ForeignKey(Asset,
+                              to_field="serial_number",
+                              null=False,
+                              on_delete=models.PROTECT)
+    current_owner = models.ForeignKey('User',
+                                      related_name='current_owner_asset',
+                                      blank=True,
+                                      null=True,
+                                      on_delete=models.PROTECT)
+    previous_owner = models.ForeignKey('User',
+                                       related_name='previous_owner_asset',
+                                       editable=False,
+                                       blank=True,
+                                       null=True,
+                                       on_delete=models.PROTECT)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+
+    class Meta:
+        verbose_name_plural = "Allocation Histories"
```

Comment:
"Allocation History" works better here

Reply:
I guess so but`histories` doesn't sound so bad does it ?
![screen shot 2018-04-30 at 12 57 02](https://user-images.githubusercontent.com/6042152/39422476-29b0fce0-4c76-11e8-8863-296f08d7fde6.png)

Reply:
it's like app changelog vs changelogs.. :)